Util/Tokenization/src/test/java/org/tribuo/util/tokens/impl/WordpieceTokenizerTest.java

```diff
@@ -0,0 +1,83 @@
+package org.tribuo.util.tokens.impl;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.tribuo.util.tokens.Tokenizer;
+import org.tribuo.util.tokens.TokenizerTestBase;
+import org.tribuo.util.tokens.impl.wordpiece.Wordpiece;
+import org.tribuo.util.tokens.impl.wordpiece.WordpiecePreprocessTokenizer;
+import org.tribuo.util.tokens.impl.wordpiece.WordpieceTokenizer;
+
+import com.oracle.labs.mlrg.olcut.util.IOUtil;
+
+public class WordpieceTokenizerTest extends TokenizerTestBase {
+
+    @Disabled
+    @Test
+    void testSoftDash() throws Exception {

```

Comment:
This doesn't seem to assert anything, should it still be here?

Reply:
removed.